src/App.php

```diff
@@ -265,13 +265,23 @@ protected function configureRoutes( Slim $slim ) {
 				$middleware['inject-user'], $middleware['set-environment'],
 			function () use ( $slim, $middleware, $routeConditions ) {
 				$slim->get( '', function() use ( $slim ) {
-					// Redirect root-URL requests to English.
-					$currentLang = $slim->i18nContext->getCurrentLanguage();
-					$slim->redirectTo( 'home', [ 'wikiLang' => $currentLang ] );
+					// See if we have a cookie indicating last version used
+					if ( isset( $_COOKIE['copypatrolLang'] ) ) {
+						$slim->redirectTo( 'home', [ 'wikiLang' => $_COOKIE['copypatrolLang'] ] );
+					} else {
+						// If no cookie, check if we support i18nContext's default language
+						$lang = $slim->i18nContext->getCurrentLanguage();
+						if ( in_array( $lang, [ 'en', 'fr' ] ) ) {
```

Comment:
Can we use a config var for this? i.e. $supportedLanguages? Or even just a member var at the top of the file would work. Something so it isn't as easy to miss when we expand the languages further.